### DISCUSSION TITLE
Hi, where i can add --twitch-access-token-param KEY=VALUE

### DISCUSSION THREAD

**Discussion by BeyondSteaLikez** [streamlink/streamlink#5439] [Q&A] [answered]
### Description

Hi, where i can add --twitch-access-token-param KEY=VALUE in the streamlink or streamlink twich gui i  honestly dont know

**Comment by BeyondSteaLikez** [comment]
i want to remove ads from specific channels how i can do that?  i am alrdy subscribed there

**Comment by bastimeyer** [comment, marked as answer]
Have a look at the documentation:
https://streamlink.github.io/cli/plugins/twitch.html

> --twitch-access-token-param

This is not what you are looking for. You want to set authentication data via [`--twitch-api-header`](https://streamlink.github.io/cli.html#cmdoption-twitch-api-header).

> where

https://streamlink.github.io/cli/config.html

**Comment by BeyondSteaLikez** [comment]
i seen the documentation but i dont know where i input that
